docs/use_cases/multimodal.md

```diff
@@ -60,6 +60,7 @@ maxdepth: 1
 GPT4-V: </examples/multi_modal/openai_multi_modal.ipynb>
 ```
 
+(multi-modal-pydantic-program)=
```

Comment:
Special markdown syntax I'm guessing?

Reply:
yah 

technically you should be able to link to a section via something like (/path/to/page.md#section) but that hasn't worked for me here